shared/src/main/java/com/example/shared/model/Player.java

```diff
@@ -86,10 +89,24 @@ public void setPointsEarned(int pointsEarned) {
         this.pointsEarned = pointsEarned;
     }
 
-    public void addCard(TrainCard trainCard) {
+    public void addTrainCard(TrainCard trainCard) {
         this.trainCards.add(trainCard);
     }
 
+    public void removeTrainCard(int number, Route.RouteColor color) {
+        //Todo: MAKE THIS WORK
+    }
+
+    public void addTicket(Ticket ticket) {
+        this.tickets.add(ticket);
+    }
+
+    public void returnedTickets(ArrayList<Ticket> returnedTickets) {
+        for (int i = 0; i < returnedTickets.size(); i++) {
```

Comment:
*More nit picking*
It would be easier to read if you generally did loops over a list using the `for (Type t : list)` construction.

Reply:
Another one of my mistakes. Feel free to update it.